.nuspec

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+    <metadata>
+        <id>Messerli.Utility</id>
+        <version>0.1.1</version>

```

Comment:
Remove version tag or make it 0.0.0

Reply:
54b0258c00e1b4e582cece5fdbc542789c37f36b

Reply:
a0584de